tests/network/bindingplugin_passt.go

```diff
@@ -371,7 +371,7 @@ EOL`, inetSuffix, serverIP, serverPort)
 			Expect(libnet.PingFromVMConsole(anotherVMI, migrateVmiAfterMigIP)).To(Succeed())
 		},
 			Entry("[IPv4]", k8sv1.IPv4Protocol),
-			Entry("[IPv6]", k8sv1.IPv6Protocol),
+			Entry("[IPv6]", k8sv1.IPv6Protocol, decorators.Quarantine),
```

Comment:
We are missing data here.
What is not working?
Where is the ticket tracing this? A ref is needed with the explanation.

Reply:
Following an offline discussion, it was decided that this test will be quarantined since we currently don't have the capacity to look into it.

Reply:
The commit message was updated to include some info based on an initial investigation of the problem.
https://github.com/kubevirt/kubevirt/issues/14492 was posted to track the quarantined test.

Reply:
@orelmisan , I hope to see this investigated as a follow up soon.
We should not leave it like this.

Any findings should be added to the issue tracking this.